Popular_Models_Guide/StableDiffusion/README.md

```diff
@@ -0,0 +1,253 @@
+pipeline and utilities.
+
+Using the TensorRT demo as a base this example contains a reusable
+[python based backend](https://github.com/triton-inference-server/backend/blob/main/docs/python_based_backends.md), [`/backend/diffusion/model.py`](backend/diffusion/model.py),
+suitable for deploying multiple versions and configurations of
+Diffusion models.
+
+For more information on Stable Diffusion please visit
+[stable-diffusion-v1-5](https://huggingface.co/runwayml/stable-diffusion-v1-5),
+[stable-diffusion-xl](https://huggingface.co/docs/diffusers/en/using-diffusers/sdxl). For
+more information on the TensorRT implementation please see the [TensorRT demo](https://github.com/NVIDIA/TensorRT/tree/release/9.2/demo/Diffusion).
+
+> [!Note]
+> This example is given as sample code and should be reviewed before use in production settings.
+
+| [Requirements](#requirements) | [Building Server Image](#building-the-triton-inference-server-image) | [Stable Diffusion v1.5](#building-and-running-stable-diffusion-v-15) | [Stable Diffusion XL](#building-and-running-stable-diffusion-xl) | [Sending an Inference Request](#sending-an-inference-request) | [Model Configuration](docs/model_configuration.md) | [Client Application](docs/client_application.md) |
+
+## Requirements
+
+The following instructions require a Linux system with Docker
+installed. For CUDA support, make sure your CUDA driver meets the
+requirements in "NVIDIA Driver" section of [Deep Learning Framework
+support matrix](https://docs.nvidia.com/deeplearning/frameworks/support-matrix/index.html).
+
+## Building the Triton Inference Server Image
+
+The example is designed based on the
+`nvcr.io/nvidia/tritonserver:24.01-py3` docker image and [TensorRT OSS v9.2.0](https://github.com/NVIDIA/TensorRT/releases/tag/v9.2.0).
+
+A set of convenience scripts are provided to create a docker image
+based on the `nvcr.io/nvidia/tritonserver:24.01-py3` image with the
+dependencies for the TensorRT Stable Diffusion demo installed.
+
+### Triton Inference Server + TensorRT OSS
+
+#### Clone Repository
+```bash
+git clone https://github.com/triton-inference-server/tutorials.git -b nnshah1-stable-diffusion --single-branch
+cd tutorials/Popular_Models_Guide/StableDiffusion
+```
+
+#### Build `tritonserver:r24.01-diffusion` Image
+```bash
+./build.sh
+```
+
+#### Included Models
+
+The `default` build includes model configuration files for
+[`stable_diffusion_1_5`](diffusion-models/stable_diffustion_1_5) and
+[`stable_diffusion_xl`](diffusion-models/stable_diffustion_xl) but the
+actual model artifacts and engine files are not included. They are
+built in a separate step.
+
+
+## Building and Running Stable Diffusion v 1.5
+
+### Start `tritonserver:r24.01-diffusion` Container
+
+The following command starts a container and volume mounts the current
+directory as `workspace`.
+
+```bash
+./run.sh
+```
+
+### Build Stable Diffusion v 1.5 Engine
+
+```bash
+./scripts/build_models.sh --model stable_diffusion_1_5
```

Comment:
I'm stepping through the tutorial and can't find this script in the container or repo folder. Is it possible it wasn't uploaded?

Reply:
thanks! uploaded.